### ISSUE TITLE
Train from scratch 

### ISSUE THREAD

**Issue by luodezhao** [laura-wang/video_repres_mas#3]
Hi, thanks for your sharing again!
I tried do train  C3D from scratch (without pretrained model), I got an accuracy of 81% which is different from what mentioned in your paper. Do you know what caused the difference?

**Comment by laura-wang** [comment]
what kind of model do you use? And 81% is the clip accuracy? 
 Actually, our number ~45% is from Du tran's C3D paper, and we use the same model as him.

**Comment by luodezhao** [comment]
Hi, thanks for your reply. The score of 81 is the train accuracy...sorry for that. 
and could you share your fine-tune code from c3d? 
Thank you~
